Il2CppInspector.CLI/Program.cs

```diff
@@ -131,16 +132,26 @@ public void Dispose()
                     Console.Error.WriteLine($"Unity path {unityPath} does not exist");
                     return 1;
                 }
-                
-                if (!File.Exists(unityPath + @"/Contents/Managed/UnityEditor.dll")) {
+
+                string editorPathSuffix = RuntimeInformation.IsOSPlatform(OSPlatform.OSX)
+                    ? @"/Contents/Managed/UnityEditor.dll"
+                    : @"\Editor\Data\Managed\UnityEditor.dll";
+
+                if (!File.Exists(unityPath + editorPathSuffix)) {
                     Console.Error.WriteLine($"No Unity installation found at {unityPath}");
                     return 1;
                 }
+                
                 if (!Directory.Exists(unityAssembliesPath)) {
                     Console.Error.WriteLine($"Unity assemblies path {unityAssembliesPath} does not exist");
                     return 1;
                 }
-                if (!File.Exists(unityAssembliesPath + @"/ScriptAssemblies/UnityEngine.UI.dll")) {
+                
+                string uiDllPath = RuntimeInformation.IsOSPlatform(OSPlatform.OSX)
+                    ? @"/ScriptAssemblies/UnityEngine.UI.dll"
```

Comment:
Is ScriptAssemblies really nested as ScriptAssemblies/ScriptAssemblies on OS X? The command line argument intends for you to pass in the ScriptAssemblies folder (see the default value for it).

Reply:
No, I was following the Readme documentation and it got a bit confusing: there are 2 lines for --unity-assemblies, I followed the first one and had to add extra ScriptAssemblies. 
```
Typical Unity project template location (specified with --unity-assemblies): C:\Program Files\Unity\Hub\Editor\20xx.y.z\Editor\Data\Resources\PackageManager\ProjectTemplates\libcache\<name-of-template>

Typical Unity script assemblies location in existing project (specified with --unity-aseemblies): X:\MyProject\Library\ScriptAssemblies
```

Reply:
Oh whoops you're absolutely right, the first line of text should have `\ScriptAssemblies` at the end of it. I will fix that. Could you modify your 2nd commit and force push - you can just use `Path.DirectorySeparatorChar' (or similar, I forget the exact name) to avoid checking the platform on the script assemblies file path check. Thanks!